### PR TITLE
Portfolio: desktop version

### PR DESCRIPTION
**Pull request by basitali111**
Making porfolio by following things:
-Add html file
-Add Css for styling
-Add linters for error checking

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,255 +1,366 @@
...
+            id="message"
+            rows="10"
+            cols="30"
+            placeholder="Write your message here"
+            required
+            maxlength="500"
+          ></textarea>
+          <input id="submit" type="submit" value="Get In Touch" />
+        </form>
+      </div>
```
Comment (ishimwezachee) [inline index.html]: - [ ]  The content of your cards should fit very well in your card as provided in the Figma design.  The size of your cards are very huge, just minimize it and fills its content with the provided data. 

Your design | Figma design
--- | ---
image  | image

File: index.html [hunk 2]
```diff
@@ -1,255 +1,366 @@
...
-                  <img src="images/css.png" alt="skill" />
-                  <p>CSS</p>
-                </li>
-                <li class="pro-item">
-                  <img src="images/javascript.png" alt="skill" />
-                  <p>Javascript</p>
-                </li>
-              </ul>
+              <a href="#"
+                ><img src="images/Group.png" alt="linkedprofilelink"
```
Comment (ishimwezachee) [inline index.html:264]: - [ ]  You should have a responsive website for both mobile and desktop. but seemed that you didn't consider this part, to make it responsive to your desktop device. 
image

**Review by ishimwezachee — CHANGES_REQUESTED** [review]
Hi @basitali111 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


### To highlight 💯 
- Good commit message ✔️ 
- No Linter errors ✔️ 
- Follow Github flow ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on @Ishimwe Zachee  in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
(Not automatically expanded because 3 MB is too large. You can [expand it anyway](https://camo.githubusercontent.com/6ac905aa3a075ef45de55758c20c6e6efb1dfdbbb0d9fd03d4718d26c5c96cde/68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f4a713832345239334a734c775a436169534c2f67697068792e676966) or [open it in a new window](https://camo.githubusercontent.com/6ac905aa3a075ef45de55758c20c6e6efb1dfdbbb0d9fd03d4718d26c5c96cde/68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f4a713832345239334a734c775a436169534c2f67697068792e676966).)

File: index.html [hunk 3]
```diff
@@ -1,255 +1,382 @@
...
+          <input
+            type="email"
+            name="email"
+            id="email"
+            placeholder="Email"
+            required
+          />
+          <textarea
+            name="message"
+            id="message"
```
Comment (Ginohmk) [inline index.html:370]: - [Optional] Please remove the excess white space on the right side of the `contact section` of your design.

##### Your design 
Screenshot from 2022-06-11 21-27-13

##### Figma Design
Screenshot from 2022-06-11 21-28-08

File: index.html [hunk 4]
```diff
@@ -1,255 +1,382 @@
...
+    <section class="contact-me">
+      <h2>Contact me</h2>
+      <p id="contact-p">
+        If you have an application you are interested in developing, a feature
+        that you need built or a project that needs coding. I’d love to help
+        with it
+      </p>
+      <div>
+        <form method="post" action="https://formspree.io/f/mnqwrjkp" id="form1">
+          <input
```
Comment (Ginohmk) [inline index.html:353]: - Please as the previous reviewer requested, the extra spaces on the desktop view on the right side should be removed so as to make your design look good and similar to the Figma design. 

##### Your design
Screenshot from 2022-06-11 21-32-38

##### Figna design
Screenshot from 2022-06-11 21-36-37

- Also please remove all extra spaces from the `contact section`, `header section`, and the `about section`., so your design looks like that of the Figma design on desktop view

**Review by Ginohmk — CHANGES_REQUESTED** [review]
Hi @basitali111  

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

keep-trying-youre-almost-there

 ## Highlights
- No linter errors :+1: 
- USed Github workflow :+1: 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by bimbolabuari — APPROVED** [review]
Hi @basitali111, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**